WooCommerce/Classes/Authentication/Navigation Exceptions/ULAccountMismatchViewController.swift

```diff
@@ -122,7 +122,9 @@ private extension ULAccountMismatchViewController {
     func configureLogOutButton() {
         logOutButton.applyLinkButtonStyle()
         logOutButton.setTitle(viewModel.logOutButtonTitle, for: .normal)
-        logOutButton.contentEdgeInsets = .zero
+        var configuration = UIButton.Configuration.plain()
+        configuration.contentInsets = .zero
+        logOutButton.configuration = configuration
```

Comment:
The `logOutButton` was configured with `applyLinkButtonStyle()`, which added a plain configuration already. Seems like setting the configuration again doesn't work: 

<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/6ba3881d-5c3f-45b0-b935-7087adea93df" width=320 />

In this case you should set the content insets to the already existing configuration:

```suggestion
        logOutButton.configuration.contentInsets = .zero
```

Also, please test your code next time before marking the PR as ready 😅